src/ORM/FieldType/DBMultiEnum.php

```diff
@@ -34,6 +45,15 @@ public function __construct($name = null, $enum = null, $default = null)
         }
     }
 
+    public function getValueForValidation(): array
+    {
+        $value = parent::getValueForValidation();
+        if (is_array($value)) {
```

Comment:
```suggestion
    public function getValueForValidation(): iterable
    {
        $value = parent::getValueForValidation();
        if (is_iterable($value)) {
```
unless there's a reason it needs to explicitly be array

Reply:
Updated